### PR TITLE
Add front channel saml docs

### PR DESCRIPTION
**Pull request by gomathyk**

### CODE REVIEW

File: en/docs/quick-starts/saml-front-channel-logout.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+----
+If you have your own application, click the button below.
+
+<a class="samplebtn_a" href="../../../guides/login/saml-front-channel-logout"   rel="nofollow noopener">I have my own application</a>
+
+----
+
+## Scenario
+
+The user is required to log in to two different applications created in a tenant, `application1` and `application2`. For convenience and security, the user should be logged out of both the applications when the user attempts to logout from either one of them.
```
Comment (DMHP) [inline en/docs/quick-starts/saml-front-channel-logout.md:14]: Do we necessarily need a tenant here? Can't we do this in the super tenant?

File: en/docs/quick-starts/saml-front-channel-logout.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+
+5.  Similar to the first application, select **localhost**, click on **Start single sign-on**, and sign in as the tenant admin. 
+
+    ![sig-in](../../assets/img/samples/sign-in-sample.png)
+
+6.  On any one of the applications click **Global Logout**.
+
+7.  You will be successfully logged out of both the applications.
+
+    ![spring-logout](../../assets/img/samples/spring-logout.png)
```
Comment (DMHP) [inline en/docs/quick-starts/saml-front-channel-logout.md:119]: Add related topics at the bottom

File: en/docs/fragments/add-new-tenant.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+!!! note
```
Comment (DMHP) [inline en/docs/fragments/add-new-tenant.md:1]: Do we need this fragment? Can't we test this in the super tenant?

File: en/docs/quick-starts/saml-front-channel-logout.md (hunk 2 above)
Comment (gomathyk) [reply en/docs/quick-starts/saml-front-channel-logout.md:113]: Added

File: en/docs/fragments/download-saml-metadata.md [hunk 4]
```diff
@@ -0,0 +1,7 @@
+1.  Login to WSO2 IS as the administrator of the tenant.
```
Comment (DMHP) [inline en/docs/fragments/download-saml-metadata.md:1]: We need to replace the fragment from the portal docs right? Can we add that standard TODO here?

File: en/docs/fragments/saml-app-front-channel-configs.md [hunk 5]
```diff
@@ -0,0 +1,8 @@
+1.   Select **Enable Response Signing**.
```
Comment (DMHP) [inline en/docs/fragments/saml-app-front-channel-configs.md:1]: We need to replace the fragment from the portal docs right? Can we add that standard TODO here?

File: en/docs/guides/login/saml-front-channel-logout.md [hunk 6]
```diff
@@ -0,0 +1,38 @@
+# Configure SAML 2.0 Front Channel logout
+
+This page guides you through [configuring SAML front channel logout](insert-link) for applications with WSO2 Identity Server. This will ensure that the user is logged out from all configured applications with a single logout request via one of the applications.
```
Comment (DMHP) [inline en/docs/guides/login/saml-front-channel-logout.md:3]: replace the insert-link with TODO:insert-link-to-concept

File: en/docs/guides/login/saml-front-channel-logout.md [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+--- 
+
+Follow the instructions to download the SAML metadata for your applications. 
+
+{! fragments/download-saml-metadata.md !}
+
+Once you have configured all your applications, access them in separate tabs in your browser. When you logout from one application, it should log you out from all the configured applications. 
+
+!!! info "Related Topics"
+    -   [Configuring SAML Front Channel Logout](insert-link)
```
Comment (DMHP) [inline en/docs/guides/login/saml-front-channel-logout.md:37]: Replace the insert-link

File: en/docs/quick-starts/saml-front-channel-logout.md [hunk 8]
```diff
@@ -0,0 +1,117 @@
+# Configure SAML 2.0 Front Channel Logout 
+
+This page guides you through configuring [SAML 2.0 front channel logout](insertlink). This is demonstrated using two versions of a sample application called Spring SAML application.
```
Comment (DMHP) [inline en/docs/quick-starts/saml-front-channel-logout.md:3]: replace insertlink with TODO:insert-link-to-concept

File: en/docs/quick-starts/saml-front-channel-logout.md [hunk 9]
```diff
@@ -0,0 +1,117 @@
...
+
+3.  Make a copy of this file and rename it as `spring-security-saml2-sample2.war` such that there are two webapps available.
+
+4.  Deploy the two webapps in the Tomcat server.
+
+---
+
+## Register two applications in WSO2 Identity Server
+
+1.  Log into WSO2 IS as the administrator.
```
Comment (DMHP) [inline en/docs/quick-starts/saml-front-channel-logout.md:70]: Shall we maintain fragments for these

Comment (sherenewso2) [reply en/docs/quick-starts/saml-front-channel-logout.md]: Log into --> Log in to

File: en/docs/quick-starts/saml-front-channel-logout.md [hunk 10]
```diff
@@ -101,14 +95,14 @@ The user is required to log in to two different applications created in a tenant
 
     ![choose-localhost](../../assets/img/samples/choose-localhost.png)
 
-3.  Login as tenant admin. You will be directed to the page with authenticated user information. 
+3.  Login as admin. You will be directed to the page with authenticated user information.
```
Comment (sherenewso2) [inline en/docs/quick-starts/saml-front-channel-logout.md:83]: Login --> Log in

"login" when its used as a noun for e.g., "Use your login credentials" and "log in" when used as a verb for e.g., "Log in to the application"

File: en/docs/quick-starts/saml-front-channel-logout.md [hunk 11]
```diff
@@ -0,0 +1,102 @@
...
+----
+
+## Configure the sample
+
+1.  Download the latest version of the Spring SAML application from [here](https://repo.spring.io/list/release/org/springframework/security/extensions/spring-security-saml/) and unzip it. This unzipped directory will be referred to as `<SAMPLE_HOME>`.
+2.  Rename the downloaded metadata file obtained from the above step as `wso2_abc.xml`.
+3.  Copy it to `<SAMPLE_HOME>/sample/src/main/resources/metadata`.
+4.  Open the `<SAMPLE_HOME>/sample/src/main/webapp/WEB-INF/securityContext.xml` file. 
+5.  Search for **bean id** with the tag as metadata (`bean id=”metadata”`). 
+6.  Under the `list` tag of the metadata, bean id add the following xml configuration and save the file. Note that this has the link to the metadata file,`wso2_abc.xml`, that we generated and renamed.
```
Comment (sherenewso2) [inline en/docs/quick-starts/saml-front-channel-logout.md:31]: of the metadata, bean id --> of the `metadata` bean id?

File: en/docs/quick-starts/saml-front-channel-logout.md [hunk 12]
```diff
@@ -0,0 +1,102 @@
...
+    ```
+
+## Deploy the sample application
+
+1.  Open a terminal from the `<SAMPLE_HOME>/sample` location and run the following command.
+
+    ```xml
+    mvn clean install
+    ```
+2.  Upon a successful project build, the `spring-security-saml2-sample.war` file can be found in the `<SAMPLE_HOME>/sample/target` directory.
```
Comment (sherenewso2) [inline en/docs/quick-starts/saml-front-channel-logout.md:61]: Upon a successful project build --> might be clearer to say "Once the sample has built successfully," 

WDYT?

File: en/docs/quick-starts/saml-front-channel-logout.md [hunk 13]
```diff
@@ -0,0 +1,102 @@
...
+
+6.  On any one of the applications click **Global Logout**.
+
+7.  You will be successfully logged out of both the applications.
+
+    ![spring-logout](../../assets/img/samples/spring-logout.png)
+    
+!!! info "Related Topics"
+    -   [Configuring SAML Front Channel Logout](TODO:insert-link-to-concept)
+    -   [Configure SAML 2.0 Front Channel Logout](../../../guides/login/saml-front-channel-logout)
```
Comment (sherenewso2) [inline en/docs/quick-starts/saml-front-channel-logout.md:102]: format:

Concept: Configuring SAML Front Channel Logout
Guide: Configure SAML 2.0 Front Channel Logout

File: en/docs/quick-starts/attribute-config-sample.md [hunk 14]
```diff
@@ -68,18 +68,21 @@ If you have your own application, click the button below.
...
 7.  Click **Get access token**. 
 
 8.  You are now successfully logged in to the sample application. You can see the values of the local attributes that were configured. You can also see the field, **Subject Claim URI** as **sub**, which contains the value of the local attribute you configured as the subject attribute URL. 
 
-    ![playground-sub](../../assets/img/samples/playground-sub.png)
+    <img src="../../assets/img/samples/playground-sub.png" alt="alt text" width="500" height="500">
+
+!!! info "Related topics" 
+    -   [Attribute Management](TODO:insert-link-to-concept)
+    -   [Request attributes](../../../guides/login/request-app-claims)
```
Comment (sherenewso2) [inline en/docs/quick-starts/attribute-config-sample.md:87]: Please change to the format:

Concept : xxxx
Guide: xxx

please check this on other pages as well

**Pull request merged**